cg/services/sample_service/sample_service.py

```diff
@@ -0,0 +1,43 @@
+from cg.services.sample_service.utils import (
+    get_cancel_comment,
+    get_confirmation_message,
+)
+from cg.store.models import User
+from cg.store.store import Store
+
+
+class SampleService:
+
+    def __init__(self, store: Store):
+        self.store = store
+
+    def cancel_sample(self, sample_id: int, user_email: str) -> list[str]:
```

Comment:
Most likely, the users will specify the internal_id rather than the database entry id.

Reply:
When you select stuff in the admin interface it is the entry id:s by default!

Reply:
Oh! Forgot that this is via the web api!